app/controllers/filetypes_controller.rb

```diff
@@ -0,0 +1,27 @@
+class FiletypesController < ApplicationController
+  def create
+    render json: filetype_json
+  rescue Hets::HetsError
+    render json: {status: 415, message: 'Media Type not supported'},
+           status: 415
+  end
+
+  protected
+
+  def filetype
+    @filetype ||= Hets::FiletypeCaller.new(HetsInstance.choose).
+      call(params[:iri])
+  end
+
+  def filetype_json
```

Comment:
Is it a convention to name a method `_json`, when the return hash is supposed to be rendered as json?